src/commands/env.rs

```diff
@@ -45,6 +48,17 @@ fn make_symlink(config: &FnmConfig) -> Result<std::path::PathBuf, Error> {
     }
 }
 
+#[derive(serde::Serialize, serde::Deserialize)]
+#[allow(non_snake_case)]
+struct EnvVars {
+    FNM_MULTISHELL_PATH: String,
+    FNM_VERSION_FILE_STRATEGY: String,
+    FNM_DIR: String,
+    FNM_LOGLEVEL: String,
+    FNM_NODE_DIST_MIRROR: String,
+    FNM_ARCH: String,
+}
```

Comment:
we can probably use `std::collections::HashMap::<String, String>` as it implements Serialize

```suggestion
type EnvVars = std::collections::HashMap::<String, String>
```

Reply:
I know that HashMap::<String,String> is a very not type-safe thing to do but it's merely an output format that will be declared in a single place within the entire system, so I am not sure we should care 🤷

Reply:
Updated to use a `HashMap` now!